lib_libwasmvm.go

```diff
@@ -73,7 +73,20 @@ func (vm *VM) StoreCode(code WasmCode, gasLimit uint64) (Checksum, uint64, error
 		return nil, gasCost, types.OutOfGasError{}
 	}
 
-	checksum, err := api.StoreCode(vm.cache, code)
+	checksum, err := api.StoreCode(vm.cache, code, true)
+	return checksum, gasCost, err
+}
+
+// SimulateStoreCode is the same as StoreCode but does not actually store the code.
+// This is useful for simulating all the validations happening in StoreCode without actually
+// writing anything to disk.
+func (vm *VM) SimulateStoreCode(code WasmCode, gasLimit uint64) (Checksum, uint64, error) {
```

Comment:
Would be nice to have a test for this new function in lib_libwasmvm_test.go

Reply:
Done